### ISSUE TITLE
Headings are not structured - Facilities Buildings

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9314]
# Motivation

SiteImprove is reporting Headings are not structured accessibility best practice issue on Facilities building pages when an alert is being displayed on the building page.

https://my2.siteimprove.com/Inspector/1005109/A11y/Page?pageId=65056259075&impmd=264ED37B65FA6186129334A508391970#/sia-r53/failed/S0YwYWpOeFU=;MDU0MGRlYTI=

# Additional Info

The alert title is being output as a h3 and the previous heading is an h1.
